pkg/serverhandler/doc.go

```diff
@@ -0,0 +1,9 @@
+// Package serverstate exports the verification harness for
+// implementing a new waypoint server protobuf (i.e. pb.WaypointServer)
+// implementations.
+//
+// Additional server implementations is not an officially supported extension
+// of Waypoint, so you do this at your own peril. It will not be supported.
+// However, this package is exported because we maintain our own private server
+// implementations for other use cases of Waypoint.
+package serverhandler
```

Comment:
Nothing in this package yet except the handler tests, but I think we'll put some interfaces in here before long to support some of the more complex tests that reach directly into state.